shared/utils/urls.ts

```diff
@@ -1,32 +1,27 @@
+import env from "../env";
 import { parseDomain } from "./domains";
 
-const env = typeof window !== "undefined" ? window.env : process.env;
-
 export function cdnPath(path: string): string {
   return `${env.CDN_URL}${path}`;
 }
 
+// TODO: HACK: if this is called server-side, it will always return false.
+// - The only call sites to this function and isExternalUrl are on the client
+//   - The reason this is in a shared util is because it's used in an editor plugin
+//     which is also in the shared code
```

Comment:
I believe this is all true - the behavior here is very unexpected so I wanted to leave an ugly comment calling it out